docs/getting-started/4-lint.md

```diff
@@ -0,0 +1,19 @@
+# Lint 
```

Comment:
I think I'd keep it in the CLI document.
This really only applies to the CLI, if you use the JS API the lining process looks differently.